Makefile

```diff
@@ -36,18 +36,30 @@ INCLUDE_FLAGS = -Iinclude -I$(DLPACK_PATH)/include -I$(DMLC_CORE_PATH)/include
 PKG_CFLAGS = -std=c++11 -Wall -O2 $(INCLUDE_FLAGS) -fPIC
 PKG_LDFLAGS =
 
+# a way to pass in extra args to the various cmake calls
+# for an example see `make build-llvm`
+CMAKE_ARGS =
+
+# a way to pass in extra args to the make calls, ex: running with 8 threads via "MAKE_ARGS='-j8'"
+MAKE_ARGS =
 
 all:
-	@mkdir -p build && cd build && cmake .. && $(MAKE)
+	mkdir -p build && cd build && cmake $(CMAKE_ARGS) .. && $(MAKE) $(MAKE_ARGS)
```

Comment:
Looks like @ was inadvertently removed here.

Reply:
good catch! This PR is very much still a PoC so some things might be not in place, but it's good to note this so it gets changed back before any later merge